src/Modal.js

```diff
@@ -466,4 +485,7 @@ Modal.Header = Header;
 Modal.Title = Title;
 Modal.Footer = Footer;
 
+Modal.TRANSITION_DURATION = 300;
+Modal.BACKDROP_TRANSITION_DURATION = 150;
```

Comment:
I bet users would ask it to be configurable. :smile: 

--
I'm fine with it anyway. :sparkle: